### ISSUE TITLE
blog 2018/03/14/omnios-bhyve-guest

### ISSUE THREAD

**Issue by mgerdts** [mgerdts/mgerdts.github.io#7]
This is for comments on

https://mgerdts.github.io/2018/03/14/omnios-bhyve-guest.html

**Comment by mounty1** [comment]
This post would be 100 times more useful with those two referenced JSON files supplied.

**Comment by mgerdts** [comment]
Yeah, sorry about that. I’ve not been using SmarOS for a few years and don’t have an example handy. If you are stuck I bet the folks you can reach on the mailing list and IRC will be very helpful. 

https://docs.smartos.org/mailing-lists-and-irc/
